gen/MotController_data.yml

```diff
@@ -61,8 +62,19 @@ c_MotController_GetDescription:
     raise NotImplementedError
 c_MotController_SetDemand:
   code: |
+    from ctre._impl.autogen.ctre_sim_enums import ControlMode
     self.hal_data['control_mode'] = mode
-    self.hal_data['value'] = demand0
+    if mode == ControlMode.PercentOutput:
+        self.hal_data['value'] = demand0 / 1023.
+    elif mode == ControlMode.Follower:
+        self.hal_data['follow_target'] = demand0 & 0xFF
+    else:
+        self.hal_data['value'] = demand0
+
+    followers = [id for id in hal_data['CAN'].keys() if self._is_follower(id)]
```

Comment:
This computation is pretty intense. Why not save some effort and just store a list of followers?